test/bench.cpp

```diff
@@ -4,87 +4,370 @@
 #include <string>
 #include <vector>
 
+#include <sstream>
+
+#include <stdio.h>
+#include <stdbool.h>
+
+#include <SDL.h>
+#include <SDL_ttf.h>
+#include <cstdio>
+#include <ctype.h>
+
+#define MAX(X, Y) (((X) > (Y)) ? (X) : (Y))
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
+
+// Define screen dimensions
+#define SCREEN_WIDTH    1200
+#define SCREEN_HEIGHT   600
```

Comment:
Screen sizes varies widely, therefore it should be set manually on launch or be detected automatically.